src/Jupyter/Visualization/StateDisplayEncoders.cs

```diff
@@ -216,6 +225,32 @@ public StateVectorToHtmlResultEncoder(IConfigurationSource configurationSource)
                     vector.SignificantAmplitudes(ConfigurationSource).Select(item =>
                     {
                         var (amplitude, basisLabel) = item;
+                        var displayPhaseAsArrows = ConfigurationSource.PhaseDisplayStyle;
```

Comment:
Small suggestion - it looks like this `displayPhaseAsArrows` variable is unused; you could delete this line.

Reply:
Gotcha! Nice catch!